src/main/java/org/gradle/profiler/CompositeProfiler.java

```diff
@@ -122,4 +122,9 @@ public void calculateGradleArgs(List<String> gradleArgs) {
             }
         };
     }
+
+    @Override
+    public boolean isCreatesStacksFiles() {
+        return delegates.stream().anyMatch(Profiler::isCreatesStacksFiles);
```

Comment:
What will happen if a composite profiler of profiler A and B, but profiler A supports stacks file, profiler B doesn't support? Will differential tool still work in this case?

Reply:
The differential flame graph generation works on the stack files only. So if any of the profiler generates the stacks, then there will be differential flame graphs as well. I don't think using multiple "actual" profilers does make much sense, and they would overwrite each others stack files currently.

Reply:
Though using something like a chrome trace profiler and async profiler is fine, and you'll get differential flames for the async profiler stacks.

Reply:
Got it.